File: app/src/main/java/cn/intret/app/picgo/model/CompareItemResolveResult.java
package cn.intret.app.picgo.model;


public class CompareItemResolveResult {

    CompareItem mCompareItem;
    /**
     * true, 表示已经按照 {@link #getCompareItem() } 标识的方法解决了冲突。
     */
    boolean resolved;


    public CompareItem getCompareItem() {
        return mCompareItem;
    }

    public CompareItemResolveResult setCompareItem(CompareItem compareItem) {
        mCompareItem = compareItem;
        return this;
    }

    public boolean isResolved() {
        return resolved;
    }

    public CompareItemResolveResult setResolved(boolean resolved) {
        this.resolved = resolved;
        return this;
    }

    @Override
    public String toString() {
        return "CompareItemResolveResult{" +
                "mCompareItem=" + mCompareItem +
                ", resolved=" + resolved +
                '}';
    }
}

File: app/src/main/java/cn/intret/app/picgo/model/image/data/CompareItemResolveResult.kt
package cn.intret.app.picgo.model.image.data


class CompareItemResolveResult {

    internal lateinit var compareItem: CompareItem
    /**
     * true, 表示已经按照 [.getCompareItem] 标识的方法解决了冲突。
     */
    internal var resolved: Boolean = false

    constructor(compareItem: CompareItem, resolved: Boolean) {
        this.compareItem = compareItem
        this.resolved = resolved
    }


    override fun toString(): String {
        return "CompareItemResolveResult{" +
                "compareItem=" + compareItem +
                ", resolved=" + resolved +
                '}'.toString()
    }

}
